d2common/d2fileformats/d2ds1/ds1.go

```diff
@@ -331,7 +661,7 @@ func (ds1 *DS1) loadSubstitutions(br *d2datautils.StreamReader) error {
 func (ds1 *DS1) setupStreamLayerTypes() []d2enum.LayerStreamType {
```

Comment:
probably could be exported, because we'd like to know about streamLayerTypes (in HellSpawner)

Reply:
`ds1.Floors`, `ds1.Walls`, `ds1.Shadows`, and `ds1.Substitutions` are now public, we should be able to get this information without a method like this being exported.